.github/workflows/ci.yaml

```diff
@@ -101,6 +101,12 @@ jobs:
       - name: Wait for ember-cli to be ready
         run: pnpm npx wait-for-localhost 4200
         working-directory: packages/host
+      - name: Start matrix
+        run: pnpm start:synapse
+        working-directory: packages/matrix
+      - name: Initialize matrix data
+        run: pnpm drop-all-dbs
+        working-directory: packages/realm-server
```

Comment:
I'm curious why you needed to add this, was something bleeding into these tests? Ideally we should not have to drop all the DB's since there are none to begin with, and the matrix server state is cleared in between each test. 

Perhaps what this really means is that you need the isolated realm for your tests like we use for the registration tests.

Reply:
Based on a discussion with @lukemelia , we decided to make the experiments realm not publicly readable. After making those changes, all matrix tests failed. I noticed this happened because the indexing failed, which was due to not having realm users when executing `pnpm start:without-matrix`. I believe I don’t need to use `drop-all-dbs`; instead, I can use `register-all`.

Reply:
hmmmm so this is really annoying because the matrix tests never touch the experiments realm. and honestly we only really need to start the base realm--not all those other realms as part of `start:all`. running register-all is fine. but I think a better solution would be to refactor `pnpm start:withou-matrix` to only start the base realm, and not start any of the other realms. probably in that case we'd want to rename the script to something more precise, like `pnpm start:services-for-matrix-tests`.

Reply:
if you opt to use `register-all` could you make a ticket to refactor `pnpm start:without-matrix` to not start unnecessary realms. then we won't loose track of the cleanup work

Reply:
I'll implement the `start:services-for-matrix-tests` script. Previously, we used the experiments realm in the matrix tests, but in this PR, I've removed it, so I believe that script can now be implemented.